src/Microsoft.AspNet.Identity/UserManager.cs

```diff
@@ -48,27 +48,23 @@ public class UserManager<TUser> : IDisposable where TUser : class
         /// <param name="contextAccessor">The accessor used to access the <see cref="HttpContext"/>.</param>
         public UserManager(IUserStore<TUser> store,
             IOptions<IdentityOptions> optionsAccessor,
-            IPasswordHasher<TUser> passwordHasher,
-            IEnumerable<IUserValidator<TUser>> userValidators,
-            IEnumerable<IPasswordValidator<TUser>> passwordValidators,
-            ILookupNormalizer keyNormalizer,
-            IdentityErrorDescriber errors,
             IServiceProvider services,
-            ILogger<UserManager<TUser>> logger,
-            IHttpContextAccessor contextAccessor)
+            ILogger<UserManager<TUser>> logger)
```

Comment:
So now we're a mishmash of constructor injection and the service locator pattern? I understand it was necessary before for ProviderTypes, but this seems almost arbitrary.